Zero-Gravity-Store-React/src/Contexts/Cart-context.js

```diff
@@ -45,9 +64,27 @@ const CartProvider = ({children})=>{
               headers: { authorization: authState.token },
             });
             cartDispatch({ type: "REMOVE_FROM_CART", payload: item });
-            console.log(res);
+            toast.success(' Remove From Cart Successfully!!', {
+              position: "bottom-right",
+              autoClose: 5000,
+              hideProgressBar: false,
+              closeOnClick: true,
+              pauseOnHover: true,
+              draggable: true,
+              progress: undefined,
+              theme: "dark",
+              });
           } catch (error) {
-            alert(error);
+            toast.error(' Something went Wrong!!', {
+              position: "bottom-right",
+              autoClose: 5000,
+              hideProgressBar: false,
+              closeOnClick: true,
+              pauseOnHover: true,
+              draggable: true,
+              progress: undefined,
+              theme: "dark",
+              });
```

Comment:
same for error toast also. Please make a common function for it and utilise it.

Reply:
Same for this too